tests/Unit/ApparentHorizons/Test_ApparentHorizonFinder.cpp

```diff
@@ -327,16 +330,30 @@ void test_apparent_horizon(const gsl::not_null<size_t*> test_horizon_called,
     std::vector<domain::CoordinateMaps::Distribution> radial_distribution{
         domain::CoordinateMaps::Distribution::Linear};
 
+    std::unique_ptr<domain::creators::time_dependence::TimeDependence<3>>
+        time_dependence;
+
+    if constexpr (std::is_same_v<Frame, ::Frame::Distorted>) {
```

Comment:
Maybe do 2 tests: 1 with UniformTranslation and 1 with Shape?   The Shape map has an Identity map for DistortedToInertial, so that map isn't really tested.  Also, the Schwarzschild test has mass=1,spin=0, so the Shape map is an identity for that test.

Reply:
Ok tested both cases now